RRemoteConfig/Poller.swift

```diff
@@ -0,0 +1,38 @@
+internal class Poller {
+    let delay: TimeInterval
+    var timer: Timer?
+
+    init(delay: TimeInterval = 60.0 * 60.0) {
+        self.delay = delay
+    }
+
+    func start(action: @escaping () -> Void) {
+        // initial fetch
+        action()
+
+        self.timer = Timer(timeInterval: self.delay, repeats: true, block: { (_) in
+            action()
+        })
+
+        // polling
+        if let timer = self.timer {
+            timer.tolerance = 0.1 * self.delay
+            DispatchQueue.main.async {
```

Comment:
By the way, how does this function when the App is put into the background? Will the timer keep running and fetch while the App is in the background?

Actually, I'm not sure exactly how Android will behave. I need to look into that 🤔

Reply:
I think (but haven't confirmed) that it will depend on the background processing entitlements configured for the app. 

When I background the sample app in the simulator the timer stops firing but when I bring it to the foreground it starts firing again.

Reply:
I made a task in the issues board to look again at this. I think it could be better to invalidate the polling timer at backgrounding and recreate it at foregrounding.

Reply:
Does that mean if the App was put in the background for an hour and then brought to the foreground, then the config would be immediately fetched? Or does the timer stop counting when put in the background?

I tested on Android and the coroutine will continue to run and fetch while the App's in the background. I'm not sure if this is the right behavior or not. I can change it to be tied to the Activity life cycle, but I think there's not a big disadvantage on Android to having it run in the background. The coroutine doesn't use many resources so the only time it will use more is while fetching, which only happens once per hour.